imod/mf6/simulation.py

```diff
@@ -97,6 +97,7 @@ def __init__(self, name):
         self.name = name
         self.directory = None
         self._initialize_template()
+        self._is_from_imod5 = False
```

Comment:
Hmm imod5 converter logic is now being mixed up with the main logic. I'm not sure this is a good thing

Reply:
Will be picked up further in https://github.com/Deltares/imod-python/issues/1222